.github/ISSUE_TEMPLATE/normative-change.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Normative Change
+about: If you are submitting a bug or a feature request that will require a normative
+  change to the specification
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Description of bug or feature request 
+
+Provide description here.
+
+## Will this require a change to CORE-AAM?
+
+If unknown, leave blank. If relevant, link bug.
+
+## Will this require a change to the ARIA authoring guide?
+
+If unknown, leave blank. If relevant, link bug.
+
+# PR tracking
```

Comment:
please leave this blank when submitting. For use as the PR moves through finalization